src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -278,25 +303,23 @@ private static IEnumerable<SuggestedActionSet> PrioritizeFixGroups(IDictionary<D
             }
 
             private IEnumerable<SuggestedActionSet> GetRefactorings(
+                IOptionService optionService,
+                IDocumentSupportsSuggestionService supportSuggestion,
                 ISuggestedActionCategorySet requestedActionCategories,
+                Workspace workspace,
                 Document document,
                 ITextSnapshot snapshot,
-                Workspace workspace,
-                IOptionService optionService,
+                SnapshotSpan range,
                 CancellationToken cancellationToken)
             {
-                // For Dev14 Preview, we also want to show refactorings in the CodeFix list when users press Ctrl+.
-                if (requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Refactoring) ||
-                    requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.CodeFix))
+                if (optionService.GetOption(EditorComponentOnOffOptions.CodeRefactorings) &&
+                    _owner._codeRefactoringService != null &&
+                    supportSuggestion.SupportsRefactorings(document) &&
+                    (requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Refactoring) ||
+                     requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Any)))
```

Comment:
Thanks for catching @heejaechang. 

When light bulb calls ```GetSuggestedActions()``` in the ""Ctrl + ." case, ```requestedActionCategories``` only contains a single category ```ANY``` (when I look under the debugger). I assumed this means that I have to check for ```requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Any)``` explicitly.

However, as you point out looks like ```.Contains()``` is smarter :) and ```requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Refactoring)``` will return ``true``` even though ```requestedActionCategories``` only contains a single category ```ANY```.

I will remove the explicit check for ```requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Any)``` above.